Tests/CompatibilitySession.Tests.ps1

```diff
@@ -0,0 +1,120 @@
+using namespace System.Management.Automation
```

Comment:
Add standard copyright notice from https://github.com/PowerShell/PowerShell/blob/fc627a1dba69ad839fcb7fe76b9aa4733e2b371f/.github/CONTRIBUTING.md#pull-request---submission

Applies to all source files